pkg/controller/admissionchecks/provisioning/storehelper.go

```diff
@@ -58,9 +57,9 @@ func (c *configHelper) FilterChecks(ctx context.Context, states []kueue.Admissio
 	return retActive, nil
 }
 
-// GetConfigForCheck - get the config used by the check identified by the checks name,
+// GetConfigForAdmissionCheck - get the config used by the check identified by the checks name,
 // nil is return if the config is missing or improperly configured (the check is not active).
-func (c *configHelper) GetConfigForCheck(ctx context.Context, checkName string) (*kueue.ProvisioningRequestConfig, error) {
+func (c *storeHelper) GetConfigForAdmissionCheck(ctx context.Context, checkName string) (*kueue.ProvisioningRequestConfig, error) {
```

Comment:
```suggestion
func (c *storeHelper) ProvReqConfigForAdmissionCheck(ctx context.Context, checkName string) (*kueue.ProvisioningRequestConfig, error) {
```

I should have said this before, sorry.

Reply:
done